### PR TITLE
Feature: added a `ProgressStepper` component.

### PR DESCRIPTION
**Pull request by MrHus**
It allows the user to indicate at which stage in a form wizard the
user currently is.

Closes: #26

### CODE REVIEW

**Comment by MrHus** [comment]
> Forgot to include styling in `_bundle`.

Done

**Comment by MrHus** [comment]
> Forgot to include styling in `_bundle`.

Done

File: packages/core/ProgressStepper/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# @42.nl/ui-core-avatar
```
Comment (jvhoven) [inline packages/core/ProgressStepper/README.md]:
```suggestion
# @42.nl/ui-core-progress-stepper
```

File: packages/core/ProgressStepper/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Installation
+
+Install the package in your project directory with:
+
+```sh
+// with npm
+npm install @42.nl/ui-core-avatar
```
Comment (jvhoven) [inline packages/core/ProgressStepper/README.md]:
```suggestion
npm install @42.nl/ui-core-progress-stepper
```

File: packages/core/ProgressStepper/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+// with yarn
+yarn add @42.nl/ui-core-avatar
```
Comment (jvhoven) [inline packages/core/ProgressStepper/README.md]:
```suggestion
yarn add @42.nl/ui-core-progress-stepper
```

File: packages/core/ProgressStepper/package.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    "@42.nl/ui-core-styling": "^1.0.5",
+    "@42.nl/ui-core-typings": "^1.0.2",
+    "classnames": "2.2.6"
+  },
+  "peerDependencies": {
+    "react": "^16.9.0",
+    "reactstrap": "^8.0.1"
+  },
+  "devDependencies": {
+    "@types/lodash.uniqueid": "4.0.6",
```
Comment (jvhoven) [inline packages/core/ProgressStepper/package.json]: Can be removed, as it is not used.

**Pull request merged**